docs/source/events/index.rst

```diff
@@ -54,6 +54,7 @@ All events are encoded as JSON. On top of the JSON built-in types the events can
 - date: ISO 8601 encoded dates either with an explicit offset or in UTC
 - interval: ISO 8601 time interval
 - money: Money value encoded as string of the floating point value and the ISO 4217 code separated by a space
+- float: A floating point number encoded as string.
```

Comment:
I think give an example of floating point number just in case someone isn't familiar or maybe the programming language they know best uses a different name for this or doesn't have a name for this, for example:

```rst
- float: A floating point number encoded as string such as 1.0 or 2.0
```

I think we don't support donations that are not whole numbers. If we actually do support donations of £2.50 then use .5 in one of the examples

Reply:
What if the API does support it, but the forms don’t? For IST 1.0 they don’t for IST 2.0 they might.

Reply:
I’ve added examples for it.